pkg/controller/operator.go

```diff
@@ -167,6 +169,9 @@ func (s *Support) Run(ctx context.Context, controller *controllercmd.ControllerC
 	// know any previous last reported time
 	go uploader.Run(ctx)
 
+	reportUpdater := externalpipeline.New(s.Controller.SmartProxy, authorizer, configPeriodic)
+	go reportUpdater.Run()
```

Comment:
it will run w/o ability to stop this goroutine. Probably this is what we want to achieve (idk)